src/v2/Components/v2/ArtworkFilter/ArtworkFilterArtworkGrid.tsx

```diff
@@ -29,17 +27,6 @@ const ArtworkFilterArtworkGrid: React.FC<ArtworkFilterArtworkGridProps> = props
     contextPageOwnerId,
   } = useAnalyticsContext()
   const context = useArtworkFilterContext()
-  const aggregations = get(props, p => p.filtered_artworks.aggregations)
-
-  /**
-   * If aggregations have not been passed as props when instantiating the
-   * <ArtworkFilter> component then populate.
-   */
-  useEffect(() => {
```

Comment:
cc @damassi since we will now always be requesting and passing these in as props, killed this.